controllers/storagecluster/job_templates_test.go

```diff
@@ -0,0 +1,59 @@
+package storagecluster
+
+import (
+	"context"
+	"testing"
+
+	configv1 "github.com/openshift/api/config/v1"
+	openshiftv1 "github.com/openshift/api/template/v1"
+	api "github.com/openshift/ocs-operator/api/v1"
+	"github.com/stretchr/testify/assert"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+)
+
+//assert(check if th)
+var platforms = append(AvoidObjectStorePlatforms,
```

Comment:
Avoid generic variable names like this, at least in this context. Since this is part of `package storagecluster`, this could conflict with other test files setting the same thing in a full test run.

That said, why are you looping over a list of platforms? I'm not aware of any special requirements where the Job Templates shouldn't exist on a particular platform, though I may be missing something of course.

Reply:
Done, Initially i was confused, i have fixed it on the new patch now, This does not require any specific restriction on platforms